test/core/rebalancingSetToken.spec.ts

```diff
@@ -863,9 +1173,9 @@ contract('RebalancingSetToken', accounts => {
         const caller = managerAccount;
         const timeFastForward = 100000;
 
-        blockchain.increaseTimeAsync(timeFastForward);
+        await blockchain.increaseTimeAsync(timeFastForward);
```

Comment:
Would really love for this whole beforeEach to be one wrapper function that encompasses all the state transitions